### PR TITLE
SW-2383 Show small values in chart when the range of values is wide

### PR DESCRIPTION
**Pull request by constanzauanini**
Add minBarLength to ensure all values (including very small ones) draw a bar in the chart

### CODE REVIEW

**Comment by karthikbtf** [comment]
Update the PR title and description if you can?

File: src/components/Plants/DashboardChart.tsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ export default function DashboardChart(props: DashboardChartProps): JSX.Element
               data: chartValues,
               barThickness: 50, // number (pixels) or 'flex'
               backgroundColor: colors,
+              minBarLength: 3,
```
Comment (karthikbtf) [inline src/components/Plants/DashboardChart.tsx:38]: that was it? nice :)

Comment (constanzauanini) [reply src/components/Plants/DashboardChart.tsx:38]: Yes :(  after fighting a lot with the code, i realize there was this prop :(

Comment (karthikbtf) [reply src/components/Plants/DashboardChart.tsx:38]: works in vercel 🎉 👏

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**